.github/workflows/ci.yaml

```
@@ -12,14 +12,17 @@ on:
 env:
   CACHE_VERSION: 1
   DEFAULT_PYTHON: 3.8
-  PRE_COMMIT_HOME: ~/.cache/pre-commit
+  PRE_COMMIT_CACHE: ~/.cache/pre-commit
```

This is not correct according to the pre-commit documentation:

<https://pre-commit.com/#managing-ci-caches>

```suggestion
  PRE_COMMIT_CACHE: ~/.cache/pre-commit
```

---

I did see that. However, as you can see [here](https://github.com/home-assistant/core/runs/1918515929?check_suite_focus=true#step:6:13) something didn't work as expected. My idea behind this change is that the default path is `~/.cache/pre-commit` anyway. So we leave it as is any just use the environment variable as helper for the cache restore action. That does work as you can see [here](https://github.com/home-assistant/core/pull/46696/checks?check_run_id=1919238267#step:6:12) and [here](https://github.com/home-assistant/core/pull/46696/checks?check_run_id=1919238267#step:8:11)

---

Alright, sounds reasonable 👍